doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -87,6 +87,7 @@ In order for the Kubernetes components to be conformant with the Reference Archi
 |`ra2.k8s.011`|Anuket profile labels|To clearly identify which worker nodes are compliant with the different profiles defined by Anuket the worker nodes must be labelled according to the following pattern: an `anuket.io/profile/basic` label must be set to `true` on the worker node if it can fulfil the requirements of the basic profile and an `anuket.io/profile/network-intensive` label must be set to `true` on the worker node if it can fulfil the requirements of the network intensive profile. The requirements for both profiles can be found in [chapter 2](./chapter02.md#22-reference-model-requirements)|||
 |`ra2.k8s.012`|Kubernetes APIs|Kubernetes [Alpha API](https://kubernetes.io/docs/reference/using-api/#api-versioning) are recommended only for testing, therefore all Alpha APIs **must** be disabled.|[req.int.api.03](./chapter02.md#22-reference-model-requirements)||
 |`ra2.k8s.013`|Kubernetes APIs|Backward compatibility of all supported GA and Beta APIs of Kubernetes **must** be supported. |[req.int.api.04](./chapter02.md#22-reference-model-requirements)||
+|`ra2.k8s.014`|Security Groups|Kubernetes **must** support NetworkPolicy feature. |[infra.net.cfg.004](chapter02.md#23-kubernetes-architecture-requirements)||
```

Comment:
Should this be "Network Policy" instead of "Security Groups"?

Since NetworkPolicy resource and NetworkPolicy APIs are Kubernetes features is this requirement necessary? For consistency as we don't list other Kubernetes features.

Reply:
`Security Groups` how it is used in [Chapter 2](https://cntt.readthedocs.io/en/latest/ref_arch/kubernetes/chapters/chapter02.html#cloud-infrastructure-software-profile-requirements)

We should list all features which have some dependencies. For example NetworkPolicy needs to be supported by all components of the networking solution.